control-plane/tests/test_service_persistence_restore_readiness.py

```
@@ -75,6 +77,51 @@ def test_service_wires_restore_readiness_internal_collaborators(self) -> None:
             "RestoreReadinessService should delegate readiness projection to a dedicated collaborator",
         )
 
+    def test_service_routes_runtime_restore_and_readiness_through_diagnostics_boundary(
+        self,
+    ) -> None:
+        store, _ = make_store()
```

Ruff (0.15.12)

[error] Undefined name `make_store`

(F821)